test/passing/tests/source-diff-friendly.ml.ref

```diff
@@ -0,0 +1,11226 @@
+[@@@foo]
+
+let ((x [@foo]) : (unit[@foo])) =
+  (() [@foo])
+  [@@foo]
+
+type t = Foo of (t[@foo]) [@foo] [@@foo]
+
+[@@@foo]
+
+module M = struct
+  type t = {l: (t[@foo]) [@foo]} [@@foo] [@@foo]
+
+  [@@@foo]
+end [@foo]
+[@@foo]
+
+module type S = sig
+  include ((module type of M [@foo]) [@foo] with type t := M.t [@foo]) [@@foo]
+
+  [@@@foo]
+end [@foo]
+[@@foo]
+
+[@@@foo]
+
+type 'a with_default =
+  ?size:int (** default [42] *) -> ?resizable:bool (** default [true] *) -> 'a
+
+type obj =
+  < meth1: int -> int  (** method 1 *) ; meth2: unit -> float  (** method 2 *) >
+
+type var =
+  [ `Foo  (** foo *)
+  | `Bar of int * string  (** bar *) ]
```

Comment:
isn't this more diff-friendly:

```ocaml
type var = [
  | `Foo  (** foo *)
  | `Bar of int * string  (** bar *)
]
```

it would also be more cconsistent with records and regular variants


e.g. when adding or reordering cases